src/pages/Fox/components/FoxWifHatHoldingLine.tsx

```diff
@@ -0,0 +1,87 @@
+import type { StackDirection } from '@chakra-ui/react'
+import { Button, Flex, HStack, Stack, Text, useColorModeValue } from '@chakra-ui/react'
+import { useTranslate } from 'react-polyglot'
+import { WalletIcon } from 'components/Icons/WalletIcon'
+import { middleEllipsis } from 'lib/utils'
+
+type FoxWifHatHoldingLineProps = {
+  accountId: string
+  accountNumber: number
+  amount: string
+  symbol: string
+  discountPercent: string
+  isClaimed?: boolean
+  onClaim?: () => void
+}
+
+const actionsPaddingLeft = { base: 10, md: 0 }
+const columnWidth = { base: '100%', md: '50%' }
+const columnDirection: StackDirection = { base: 'column', md: 'row' }
+const columnAlignItems = { md: 'center' }
+const columnJustifyContent = { md: 'space-between' }
+const columnSpacing = { base: 4, md: 48 }
+
+export const FoxWifHatHoldingLine = ({
+  accountId,
+  accountNumber,
+  amount,
+  symbol,
+  discountPercent,
+  isClaimed,
+  onClaim,
+}: FoxWifHatHoldingLineProps) => {
+  const textColor = useColorModeValue('gray.500', 'gray.400')
+  const translate = useTranslate()
+
+  return (
+    <Stack
+      width='full'
+      px={6}
+      py={4}
+      spacing={columnSpacing}
+      direction={columnDirection}
+      alignItems={columnAlignItems}
+      justifyContent={columnJustifyContent}
+    >
+      <Flex width={columnWidth} alignItems='center' justifyContent='space-between'>
+        <HStack spacing={4}>
+          <WalletIcon color={textColor} boxSize={6} />
+          <Flex direction='column' alignItems='flex-start'>
+            <Text fontWeight='bold' fontSize='sm'>
+              {middleEllipsis(accountId)}
+            </Text>
+            <Text color={textColor} fontSize='xs'>
+              {translate('accounts.accountNumber', { accountNumber })}
```

Comment:
q: can we guard this against non-multi account wallets?

 i.e if there is only one account for the chain of said `accountId`, it may not make sense for users to only see `Account #0` (which can actually *add* confusion for MM-like wallets, as what we call account 0 may not actually be the 0th account in users' wallet)